ush/mesoscale/mesoscale_stats_grid2obs_var_defs.py

```diff
@@ -421,21 +421,21 @@
                 'var1_fcst_name': 'CAPE',
                 'var1_fcst_levels': 'P0-90',
                 'var1_fcst_thresholds': 'ge250,ge500,ge1000,ge1500,ge2000,ge3000,ge4000',
-                'var1_fcst_options': 'cnt_thresh = [ >0 ];',
+                'var1_fcst_options': 'cnt_thresh = [ NA ]; cnt_logic = INTERSECTION;',
                 'var1_obs_name': 'MLCAPE',
                 'var1_obs_levels': 'L0-90000',
                 'var1_obs_thresholds': 'ge250,ge500,ge1000,ge1500,ge2000,ge3000,ge4000',
-                'var1_obs_options': 'cnt_thresh = [ >0 ]; cnt_logic = UNION;',
+                'var1_obs_options': 'cnt_thresh = [ >0 ]; cnt_logic = INTERSECTION;',
```

Comment:
Just to clarify, this changes makes it so the fcst **and**  obs meet the threshold, where union could have been either or, correct?

Reply:
This change is an exact copy of PR 567 in Marcel's PR, which I think corrects the CAPE calculations.  I'll have to get a better understanding of it, but I do believe you are correct.

Reply:
@AndrewBenjamin-NOAA That is what is was before. It would verify where obs and fcst were both >0. Now the fcst has no threshold and obs >0.